infrastructure/app/chart/energy-apps/templates/vault_secrets.yaml

```diff
@@ -0,0 +1,29 @@
+{{- range  $name, $config := .Values.vaultSecrets -}}
+{{ if $config.enabled }}
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: vault-secret-{{ $name }}
+  labels:
+    {{- include "casebook.labels" $ | nindent 4 }}
```

Comment:
I'm assuming this should be `energy-apps`? And updating/deleting line 18 which also references casebook? 😄

Reply:
Oooh it's like this currently! I don't think this is used so I'm going to try an delete it.  The app doesn't store any secrets in Vault.